.github/workflows/rector.yml

```
@@ -0,0 +1,66 @@
+      matrix:
+          os: [ubuntu-latest]
+          php-version: ['8.5']
+
+    steps:
+      - name: Checkout.
+        uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7.0.1
+        with:
+          persist-credentials: false
+
+      - name: Setup PHP Environment.
+        uses: ./.github/actions/php-setup
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/php-setup": line 6: unexpected key "type" for definition of input "tools"

(action)